Makefile

```diff
@@ -46,6 +48,18 @@ test_$(1): compile $(1)
 	$(Q) cd lib/$(1) && ../../bin/elixir -r "test/test_helper.exs" -pr "test/**/*_test.exs";
 endef
 
+define write_SOURCE_DATE_EPOCH
```

Comment:
It seems that our convention here is to have those all in uppercase, so what about WRITE_SOURCE_DATE_EPOCH and READ_SOURCE_DATE_EPOCH?